gensrc/thrift/InternalService.thrift

```diff
@@ -123,6 +123,11 @@ enum TOverflowMode {
   REPORT_ERROR = 1;
 }
 
+struct TQueryQueueOptions {
+  1: optional bool enable_query_queue;
```

Comment:
enable_global_query_queue ?

Reply:
Good suggestion. `enable_global_query_queue ` is better, because `enable_global_query_queue` and `enable_group_level_queue` can be orthometric in this way.